src/t.jl

```diff
@@ -159,12 +159,14 @@ function EqualVarianceTTest(nx::Int, ny::Int, mx::Real, my::Real, vx::Real, vy::
 end
 
 """
-    EqualVarianceTTest(x::AbstractVector{T<:Real}, y::AbstractVector{T<:Real})
+    EqualVarianceTTest(x::AbstractVector{T<:Real}, y::AbstractVector{T<:Real}, μ0::Real=0)
 
 Perform a two-sample t-test of the null hypothesis that `x` and `y` come from distributions
-with equal means and variances against the alternative hypothesis that the distributions
-have different means but equal variances.
+with equal variances and mean difference μ0 (defaults to 0) against the alternative hypothesis
+that the distributions have equal variances and a mean difference different from μ0.
```

Comment:
It seems important to mention the order of the subtraction:
```suggestion
with equal variances and a difference between their means equal to `μ0`
(i.e. `mean(X) - mean(Y) == μ0`) against the alternative hypothesis that
the distributions have equal variances and a difference between their means
different from `μ0`. By default  `μ0 = 0`, that is the null hypothesis
is that means are equal.
```

Reply:
Would you be okay with rewording from "
By default  `μ0 = 0`, that is the null hypothesis is that means are equal."
to
"By default  the null hypothesis is that the means are equal, meaning `μ0 = 0`"

Reply:
Yeah sure.